nodeup/pkg/model/docker.go

```diff
@@ -571,6 +612,28 @@ var dockerVersions = []dockerVersion{
 		Hash:          "0a1325e570c5e54111a79623c9fd0c0c714d3a11",
 		Dependencies:  []string{"libtool-ltdl", "libseccomp", "libcgroup"},
 	},
+
+	// 18.06.2 - CentOS / Rhel7 (two packages)
+	{
+		DockerVersion: "18.06.2",
+		Name:          "container-selinux-2",
+		Distros:       []distros.Distribution{distros.DistributionRhel7, distros.DistributionCentos7},
+		Architectures: []Architecture{ArchitectureAmd64},
+		Version:       "18.06.2.ce",
```

Comment:
Here you should have the `container-selinux` version `2.68`. If you let it as it is `kops-configuration` will keep retrying to install and will fail causing the Node to be lost.

Reply:
Thanks for spotting - addressed in #6492